game/games.go

```diff
@@ -0,0 +1,59 @@
+package game
+
+import (
+	"encoding/json"
+	"errors"
+	"fmt"
+	"net/http"
+	"strconv"
+)
+
+type Games struct {
+	games []Game
+	seq   int
+}
+
+func NewGames() Games {
+	return Games{seq: 0}
+}
+
+func (g *Games) AddGame(name string) {
+	g.games = append(g.games, NewGame(name))
+}
+
+func (g Games) findGame(id int) (*Game, error) {
+	for _, gm := range g.games {
+		if gm.ID == id {
+			return &gm, nil
+		}
+	}
+	return nil, errors.New("Couldn't find game")
+}
+
+func (g Games) Index(w http.ResponseWriter, r *http.Request) {
+	payload, err := json.Marshal(g.games)
+	if err != nil {
+		fmt.Println("Error marshaling games data")
+	}
+
+	fmt.Fprintf(w, string(payload))
+}
+
+func (g Games) Show(w http.ResponseWriter, r *http.Request) {
+	id, err := strconv.ParseInt(r.URL.Path[len("/games/"):], 10, 32)
```

Comment:
We should look at replacing our muxer with Gorilla, or something similar. It'll be a lot easier to declare and extract route params.

Reply:
Agreed. I wanted to start with std library but rather than augmenting it with our own param extraction it'd make sense to use a built one